bananas_server/index/local.py

```diff
@@ -151,6 +155,7 @@ def _read_content_entry_version(self, content_type, unique_id, data, md5sum_mapp
         size += 1
         for tag in data.get("tags", []):
             size += len(tag) + 2
+        size += len(upload_date) + 2
```

Comment:
We need to check if there is any production BaNaNaS entry now that exceeds packet size. I believe `--validate` does this, but I am not really sure, I have to admit. Just something we need to take care of before merging :)

Reply:
I agree, is this something you could try running in the server? or I can do somehow? since I don't have the files it only loads 4 of them (the ones you sent :-))

Reply:
Tnx for the reminder, I forgot; all current content checks out fine, so that is good :)

That does remind me, we also need to change this in bananas-api:
https://github.com/OpenTTD/bananas-api/blob/7e9f4cf8c79cd6954607c8e43595d3c4d27e516e/bananas_api/new_upload/session_validation.py#L98

It ensures nobody can accidentally upload a blob bigger than what is allowed :)

Reply:
Sounds good! I'll propose those changes as well, keep u posted

Reply:
There you go. https://github.com/OpenTTD/bananas-api/pull/85